grid/recruitment/create-lead/maps/workable.suma

```diff
@@ -290,3 +315,27 @@ operation CheckJobExists {
     }
   }
 }
+
+operation CheckSupportedFileTypes {
+  cv = args.cv || {}
+
+  return if(cv.data === undefined || cv.data === null) {}
+
+  fail if(cv.fileName === undefined || cv.fileName === null) {
+    title = "Missing CV file name"
+    detail = "CV file name is required."
+    code = "CVFileNameRequired"
+  }
+  
+  fail if(
```

Comment:
I think there should be a link to the docs in which the extensions are listed, in case they ever change for example

Reply:
added